docs/src/literate/hyperelasticity.jl

```diff
@@ -18,36 +18,44 @@
 # In this example we will solve a problem in a finite strain setting using an
 # hyperelastic material model. In order to compute the stress we will use automatic
 # differentiation, to solve the non-linear system we use Newton's
-# method, and for solving the Newton increment we use conjugate gradient.
+# method, and for solving the Newton increment we use conjugate gradients.
 #
-# The weak format is expressed in terms of the first Piola-Kirchoff stress ``\mathbf{P}``
+# The weak form is expressed in terms of the first Piola-Kirchoff stress ``\mathbf{P}``
 # as follows: Find ``u \in \mathbb{U}`` such that
 #
 # ```math
-#  \int_\Omega [\delta \mathbf{u} \otimes \nabla] : \mathbf{P}(\mathbf{u})\ \mathrm{d}\Omega =
-# \int_\Omega \delta \mathbf{u} \cdot \mathbf{b}\ \mathrm{d}\Omega + \int_{\Gamma^\mathrm{N}}
-# \mathbf{u} \cdot \mathbf{t}\ \mathrm{d}\Gamma
+# \int_{\Omega} \nabla_{\mathbf{X}} \delta \mathbf{u} : \mathbf{P}(\mathbf{u})\ \mathrm{d}\Omega =
```

Comment:
Perhaps define `\nabla_{\mathbf{X}} \bullet` somewhere since it was discussed?

Reply:
Sure.